indico/modules/events/surveys/models/submissions.py

```diff
@@ -75,6 +75,10 @@ class SurveySubmission(db.Model):
     )
 
     @property
+    def locator(self):
+        return dict(submission_id=self.id, **self.survey.locator)
```

Comment:
other way round would be much nicer.

    dict(self.survey.locator, submission_id=self.id)